core/iwasm/interpreter/wasm_loader.c

```diff
@@ -3601,17 +3629,34 @@ wasm_loader_unload(WASMModule *module)
         }
     }
 #endif
-
+#if WASM_ENABLE_DEBUG_ENGINE != 0
+    WASMFastOPCodeNode * fast_opcode = bh_list_first_elem(&module->fast_opcode_list);
+    while(fast_opcode) {
+        WASMFastOPCodeNode * next = bh_list_elem_next(fast_opcode);
+        wasm_runtime_free(fast_opcode);
+        fast_opcode = next;
+    }
+#endif
     wasm_runtime_free(module);
 }
-
+#if WASM_ENABLE_DEBUG_ENGINE != 0
+bool
+wasm_loader_find_block_addr(WASMExecEnv *exec_env,
```

Comment:
Maintaining two function prototypes takes lot of effort, how about adding exec_env persistent, and ignoring it when it isn't used, so we don't need to add so many macros to control it.